demo/src/assets/styles/main.scss

```diff
@@ -278,3 +278,15 @@ tr.status-sent {
   border: 1px solid #efeff1;
   cursor: not-allowed;
 }
+
+#instagram-preview {
+  text-align: center;
+
+  .instagram-preview__iframe {
+    padding-top: 15px;
+  }
+
+  .instagram-media-rendered {
+    display: inline-block !important;
+  }
+}
```

Comment:
let's put a stylesheet in the directory of the component and then import, ref how divider-block does it. also would prefer not to use `!important` if we don't have to

Reply:
Used Flexbox instead of `!important`.